public/index.html

```diff
@@ -5,7 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <meta name="referrer" content="same-origin">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <meta name="description" content="qBittorrent Web UI" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-title" content="qBt Web">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    
+    <!-- Chrome, Opera, and Firefox OS -->
+    <meta name="theme-color" content="#4d8ad5" />
+    <!-- Windows Phone -->
+    <meta name="msapplication-navbutton-color" content="#4d8ad5" />
+    
+    <link rel="icon" type="image/x-icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="apple-touch-icon" sizes="180x180" href="<%= BASE_URL %>apple-touch-icon-180x180.png" />
```

Comment:
Please remove `sizes`. see https://webhint.io/docs/user-guide/hints/hint-apple-touch-icons/#examples-that-pass-the-hint

Reply:
Why? It should always be there.

Reply:
Read the article please.